### PR TITLE
Delay load dependencies to improve load time

### PR DESCRIPTION
**Pull request by nikhilkh**
This fixes issue #80

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,13 +3,7 @@ var spawn = require('child_process').spawn;
 var path = require('path');
 var format = require('util').format;
 var Configstore = require('configstore');
```
Comment (sindresorhus) [inline index.js]: I would also inline `configstore`. From earlier measurements I've found it to be slow to load too.

**Comment by sindresorhus** [comment]
I think it would be better to use https://github.com/sindresorhus/lazy-req so we could still declare all imports at the top.

**Comment by nikhilkh** [comment]
@sindresorhus thanks for the review - I've filed another PR with an alternate implementation: #82. Please help review - I'm fine with whichever one you want to use. Once this is accepted and updated on npm - we'll take a dependency on it @ https://github.com/apache/cordova-cli

**Comment by SBoudrias** [comment]
Closing this one in favor of the other PR.

**Pull request closed without merging**